src/nupic/types/Types.hpp

```diff
@@ -123,8 +123,10 @@ typedef std::size_t Size;
  */
 #ifdef NTA_DOUBLE_PRECISION
   typedef Real64 Real;
+  const int FLOAT_PRECISION = 8; // bytes
```

Comment:
nitpick: try using the `sizeof` macro: `sizeof( Real ) -> number of bytes`

Reply:
I agree. Didn't like this solution either. Thanks for the solution.
I'll your solution to use.

Reply:
Okay. I have restored the original Types.hpp and used sizeof in py_SpacialPooler.cpp